react-native/android/src/main/java/io/realm/react/RealmReactModule.java

```diff
@@ -125,8 +125,29 @@ public void onCatalystInstanceDestroy() {
     }
 
     private static boolean isRunningOnEmulator() {
-        // Check if running in Genymotion or on the stock emulator.
-        return Build.FINGERPRINT.contains("vbox") || Build.FINGERPRINT.contains("generic");
+         return Build.FINGERPRINT.startsWith("generic")
```

Comment:
Can you add a comment explaining where this code comes from?  I think it would be good to find a way to derive this from `react-native-device-info`, but I realize that may not be so practically done.

Reply:
I added a comment in the code explaining where this list came from.